docs/faq.md

```diff
@@ -202,6 +202,60 @@ jib.from.image = 'custom-base-image'
 
 We currently support adding a custom directory with an **incubating** feature. This feature may change in later versions. If your application needs to use custom files, place them into the `src/main/jib` folder. Files placed here will be added to the filesystem of the container. For example, `src/main/jib/foo/bar` would add `/foo/bar` into the container filesystem.
 
+### <a id="i-want-more-control-over-which-files-to-place-in-the-extra-directory"></a>I want more control over which files to place in the extra directory/where they end up in the container
```

Comment:
Is the anchor tag required for some reason?

Missing an ending period.

Reply:
I couldn't get the inferred link to work for some reason (maybe because of the '/' or title length?) so I just made it explicit.

Reply:
Try `directory-where` (i.e., use `-` instead of `/`).

Reply:
Sorry, seems like `directorywhere` (nothing between "directory" and "where").